common/pom.xml

```diff
@@ -37,6 +38,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
oh, shoot,  this is the other new dependency.  This is needed by the spring-security WebSecurityConfigurerAdapter.  Think I should switch it to "provided" ?